Libplanet.Tests/Libplanet.Tests.csproj

```diff
@@ -18,13 +16,11 @@
     <AdditionalFiles Include="..\stylecop.json" />
   </ItemGroup>
 
-  <PropertyGroup Condition=" '$(MSBuildRuntimeType)'!='Mono' And
-                             '$(MSBuildVersion)'&gt;='16.3.0'">
+  <PropertyGroup Condition=" '$(MSBuildRuntimeType)'!='Mono' And '$(MSBuildVersion)'&gt;='16.3.0'">
     <TargetFramework>netcoreapp3.0</TargetFramework>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(MSBuildRuntimeType)'=='Mono' And
-                             '$(BuildingByReSharper)'!='true'">
+  <PropertyGroup Condition=" '$(MSBuildRuntimeType)'=='Mono' And '$(BuildingByReSharper)'!='true'">
     <TargetFramework>net47</TargetFramework>
   </PropertyGroup>
```

Comment:
As Rider became based on .NET Core (was Mono before), I guess we don't need this `<PropertyGroup>` anymore… 🤔

Reply:
Isn't the `<PropertyGroup>` for Mono testing?

Reply:
IIRC the second condition `'$(BuildingByReSharper)'!='true` purposed to be *not* run on Rider, because at that time under Rider `$(MSBuildRuntimeType)` was set to `'Mono'`.